src/block/block_compact.c

```diff
@@ -443,13 +463,14 @@ __wt_block_compact_skip(WT_SESSION_IMPL *session, WT_BLOCK *block, bool *skipp)
 
     /*
      * We do compaction by copying blocks from the end of the file to the beginning of the file, and
-     * we need some metrics to decide if it's worth doing. Ignore small files, and files where we
-     * are unlikely to recover 10% of the file.
+     * we need some metrics to decide if it's worth doing. Ignore files smaller than the configured
+     * threshold, and files where we are unlikely to recover 10% of the file.
```

Comment:
I think we should stick to one term for target/threshold.

Reply:
We can also remove the "and files where we are unlikely to recover 10% of the file."